src/please.go

```diff
@@ -350,6 +350,7 @@ var opts struct {
 		Changes struct {
 			Since            string `short:"s" long:"since" default:"origin/master" description:"Revision to compare against"`
 			IncludeDependees string `long:"include_dependees" default:"none" choice:"none" choice:"direct" choice:"transitive" description:"Include direct or transitive dependees of changed targets."`
+			Level            int    `long:"level" default:"0" description:"Levels of the dependencies of changed targets."`
```

Comment:
Can you document `-1` here? 

Also we're currently in a major release cycle and I think I'd prefer not to have 2 flags that do basically the same thing. We were burnt last time we outright removed a flag so we should probably keep the flag here but have it do nothing. 

I think it makes sense to have something that looks like: `plz query change --dependency_depth=-1`

Reply:
Okay as discussed, I think removing this flag right now will cause a lot of issues because this command is used across commit versions. If we remove it, we won't be able to query changes for the commit that upgrades Please...